docs/toml-schema.md

```diff
@@ -209,6 +209,37 @@ excluded-people = [
     "rylev",
 ]
 
+# Define the Zulip streams used by the team
+# It's optional, and there can be more than one.
+#
+# This will remove anyone who isn't in the team from the stream
+# so it should primarily be used for private streams.
```

Comment:
I don't see a `private` boolean or similar. Does that mean that all zulip-streams are private?
if yes, should we change this comment?
```suggestion
# so it should only be used for private streams.
```

Reply:
I'll update the comment - there's no `private` boolean at the moment because all this will do initially is add and remove members, it can be extended later to create streams (where we'd want to specify a stream is private) and to only add members (so it makes sense to use it on a public stream).